network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -263,7 +263,7 @@ <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
     <div class="container">
       <div class="row">
         <div class="{{section_class}} mt-5 pt-3 bg-white product-section-padding-x">
-          {% if product.updates|length > 0 or coralTalkServerUrl %}
+          {% if product.updates|length > 0 %}
```

Comment:
this is the "old" product_page.html, which is no longer being used. Deleting the coral talk code still made sense, though.